TAO/tao/Monitor/TAO_Monitor.pc.in

```diff
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: TAO_Monitor
 Description: TAO Monitor Library
-Requires: TAO_PortableServer, TAO_ETCL_Parser, ACE_MonitorControl
+Requires: TAO_PortableServer, ACE_ETCL_Parser
```

Comment:
ACE_MonitorControl should be there, looks a pc.in is missing in https://github.com/DOCGroup/ACE_TAO/tree/master/ACE/ace/Monitor_Control